run_brer/run_config.py

```diff
@@ -408,9 +411,11 @@ def __converge(self, tpr_file=None, **kwargs):
         for name in self.__names:
             current_alpha = self.run_data.get('alpha', name=name)
             current_target = self.run_data.get('target', name=name)
-            self._logger.info("Plugin {}: alpha = {}, target = {}".format(name,
-                                                                          current_alpha,
-                                                                          current_target))
+            current_stop_called = self.run_data.get('stop_called', name=name)
```

Comment:
Oh. Oops. Didn't notice this before. "stop_called" is only available on the LinearStopRestraint potential, which is only used in the convergence phase. We will need different checks in the training and production phases.

Reply:
Removed `current_stop_called` from production but left in convergence function